templates/.github/workflows/run-synthetics.yml

```diff
@@ -8,19 +8,20 @@ on:
     - cron: '* 0 * * *'
 jobs:
   test:
+    defaults:
+      run:
+        working-directory: ./advanced-examples
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
-      - uses: actions/setup-node@v2
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
         with:
-          node-version: 16
+          node-version: 18
```

Comment:
Synthetics uses node 16 by default, it would be better to use the same here.

Reply:
I'm thinking we should update to 18 soon anyway, and it does seem to work. Could I put that into this PR as well? I wouldn't mind updating the heartbeat images etc. too to help keep us on the current LTS version.